docs/src/main/asciidoc/_elytron/Elytron_and_Java_Authorization_Contract_for_Containers-JACC.adoc

```diff
@@ -87,3 +87,5 @@ to enable JACC you can execute a command as follows:
 ----
 [standalone@localhost:9990 /] /subsystem=ejb3/application-security-domain=my-security-domain:write-attribute(name=enable-jacc,value=true)
 ----
+
+The `application-security-domain` resource of the EJB3 subsystem also has an option `legacy-compliant-principal-propagation`. If this is set to true and there is no incoming run-as identity, then the principal of the local unsecured bean is the current authenticated identity. This was the case for legacy PicketBox security. If this attribute is set to false, the behaviour will comply with the Elytron's previous behaviour and if there is no incoming run-as identity then the principal of the local unsecured bean is anonymous. This attribute is optional and default value is true.
```

Comment:
The text here looks good. Maybe instead of adding it in this document, we could add it to the EJB3 section here:

https://github.com/wildfly/wildfly/blob/master/docs/src/main/asciidoc/_elytron/Using_WildFly_Elytron_with_WildFly.adoc

Reply:
@fjuma Fixed. Thanks